src/main/java/org/apache/sling/dynamicinclude/package-info.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-@Version("4.0.0")
+@Version("4.0.1")
```

Comment:
I think this was needed due to the fields added to `Configuration` being package-private. Can you check after making them private if the version bump is still needed? (It might be due to the annotation, in which case leave it as-is).